apps/site/assets/ts/schedule/components/__trips.d.ts

```diff
@@ -29,12 +30,14 @@ export interface TripInfo {
   status: string;
   duration: number;
   fare: Fare;
+  route_type: number;
```

Comment:
We probably want to use the [`RouteType` type](https://github.com/mbta/dotcom/blob/dbffbc3496dc08d25a7c5733e806aab3fd96b140/apps/site/assets/ts/__v3api.d.ts#L100) instead of number here.